research/delf/delf/python/training/model/resnet50.py

```diff
@@ -456,27 +457,4 @@ def log_weights(self):
           logging.info(weights)
       else:
         logging.info('Layer %s does not have inner layers.',
-                     layer.name)
-
-
-def gem_pooling(feature_map, axis, power, threshold=1e-6):
-  """Performs GeM (Generalized Mean) pooling.
-
-  See https://arxiv.org/abs/1711.02512 for a reference.
-
-  Args:
-    feature_map: Tensor of shape [batch, height, width, channels] for
-      the "channels_last" format or [batch, channels, height, width] for the
-      "channels_first" format.
-    axis: Dimensions to reduce.
-    power: Float, GeM power parameter.
-    threshold: Optional float, threshold to use for activations.
-
-  Returns:
-    pooled_feature_map: Tensor of shape [batch, channels].
-  """
-  return tf.pow(
-      tf.reduce_mean(tf.pow(tf.maximum(feature_map, threshold), power),
-                     axis=axis,
-                     keepdims=False),
-      1.0 / power)
+                     layer.name)
```

Comment:
Similarly to the comment above, the missing new line at the end of the file is inconsistent with our internal codebase. Is this expected?